lib/resources/storage/web_storage.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:firebase/firebase.dart';
+import 'package:flutter/cupertino.dart';
+
+class FireStorageService extends ChangeNotifier {
+  FireStorageService() {
+    initializeApp(
```

Comment:
What's the point of this? Is it ever even called? The only method in this class is static, I don't see the constructor being called anywhere.